guessNumb.js

```diff
@@ -0,0 +1,84 @@
+//--------------------------READLINE------------------------------//
+
+const readline = require("readline");
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+//-------------------------AYSNC START----------------------------//
+
```

Comment:
Hoisting only works for a certain type of function (declaration) and not the other (expression). While it is viable to hoist your function declaration invocation (as you did here), I would encourage you to not utilize hoisting until you are farther along in your JS knowledge and your choice is more informed!